src/constants.rs

```diff
@@ -0,0 +1,26 @@
+pub(crate) mod v1_0 {
+    pub const SYSTEM_SYMBOLS: &[&str] = &[
+        "$0",                       // $0
+        "$ion",                     // $1
+        "$ion_1_0",                 // $2
+        "$ion_symbol_table",        // $3
+        "name",                     // $4
+        "version",                  // $5
+        "imports",                  // $6
+        "symbols",                  // $7
+        "max_id",                   // $8
+        "$ion_shared_symbol_table", // $9
+    ];
+
+    pub(crate) mod system_symbol_ids {
+        pub const ION: usize = 1;
```

Comment:
Are we leaving symbol 0 for later?

Reply:
Right, there's currently no code in place to handle SID zero. I hope to add `ion-tests`' bad files soon though, and I'm sure `$0` will lead to lots of failures. 😛